engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -132,6 +133,16 @@ private ConfigurationImpl() {
 		this.addConstraintDefinitionContributor( defaultConstraintDefinitionContributor );
 	}
 
+	private ValidatedValueUnwrapper<?> createJavaFXUnwrapperClass(TypeResolutionHelper typeResolutionHelper) {
+		try {
+			Class<?> jfxUnwrapperClass = run( LoadClass.action( JFX_UNWRAPPER_CLASS, getClass().getClassLoader() ) );
+			return (ValidatedValueUnwrapper<?>) ( jfxUnwrapperClass.getConstructor( TypeResolutionHelper.class ).newInstance( typeResolutionHelper ) );
```

Comment:
The current code might just behave like the old code. One would need 
1. something like a `ClassFoundIn` which instead of a `Class` as in `LoadClass` returns a classloader (the one which successfully loaded the class or null)
2. this classloader should then be used to load the unwrapper, but it needs a modified version of `LoadClass` to prevent the special handling of the loading of validator classes

Given 1 +  2, there might then still be the question around the `TypeResoltionHelper` @gunnarmorling mentioned.